### PR TITLE
Update Kube-ovn to 1.10.6

### PR DESCRIPTION
**Pull request by zhouqiu0103**
### What type of PR is this?

/kind feature
### What this PR does / why we need it:
- Update Kube-ovn to 1.10.6
-  Adjusting the structure of the Kube-ovn configuration file:split a simple file `network-plugin.yaml`  into three files `kube-ovn-crd.yaml`,`ovn.yaml` and `kube-ovn.yaml`.
### Which issue(s) this PR fixes:

Fixes #

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/plugins/network/templates/kubeovn.go [hunk 1]
```diff
@@ -1184,7 +1185,10 @@ spec:
     kind: HtbQos
     shortNames:
       - htbqos
----
+`)))
+
+	OVN = template.Must(template.New("ovn.yaml").Parse(
+		dedent.Dedent(`---
 {{ if .DpdkMode }}
```
Comment (24sama) [inline pkg/plugins/network/templates/kubeovn.go:1192]: I see the DpdkMode has been removed on the above, so is this piece of code still needed?

Comment (zhouqiu0103) [reply pkg/plugins/network/templates/kubeovn.go:1192]: DpdkMode was in this location before the network-plugin.yaml was split

Comment (24sama) [reply pkg/plugins/network/templates/kubeovn.go:1192]: OK, I got it.

**Pull request merged**